.env.example

```diff
@@ -0,0 +1,66 @@
+# Change this for production
+SECRET_KEY=ChangeMeToARandomString
+
+# Turn this to True for development.
+DEBUG=False
+
+# This is used by the tracker mailer to determine where to redirect
+# registration and password resets to.
```

Comment:
Eventually this can be ditched entirely, I think that code is old enough that it predates the helper method to build URLs based on the current request.